packages/jest-utils/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": ["plugin:@empathyco/x/all"],
+  "rules": {
+    "@typescript-eslint/restrict-template-expressions": ["off", { "allowAny": true }],
```

Comment:
I've noticed that both problems come from: x/packages/jest-utils/src/jest-utils.ts.
Isn't it preferable to disable both rules at least just for that file ?